app/src/components/ui/Hyperlink.tsx

```diff
@@ -0,0 +1,71 @@
+import { cva, type VariantProps } from "cva";
+import Link from "next/link";
+import {
+  type AnchorHTMLAttributes,
+  // type ClassAttributes,
+} from "react";
+// import { FiRotateCw } from "react-icons/fi";
+import { twMerge } from "tailwind-merge";
+
+const HyperLinkStyles = cva(
+  "transition-all font-inter px-1 py-0.5 border-2 rounded border-transparent text-base font-semibold underline underline-offset-2",
+  {
+    variants: {
+      intent: {
+        default: "text-brand-600 hover:text-brand-800 active:text-brand-900 focus:text-brand-600 focus:border-brand-600",
+        disabled: "text-disabled-400",
+      },
+      fullWidth: {
+        true: "w-full",
+      },
+    },
+    defaultVariants: {
+      intent: "default",
+    },
+  }
+);
+
+type HyperLinkProps = {
+  isLoading?: boolean;
+} & AnchorHTMLAttributes<HTMLAnchorElement> &
+  VariantProps<typeof HyperLinkStyles>;
```

Comment:
do we need `isLoading` as a prop here? i don't see it being used in the `HyperLink` component.